### PR TITLE
[A4] dev/signUpDisplay: Add signUp.form

### PR DESCRIPTION
**Pull request by hzl616**

### CODE REVIEW

File: src/main/java/org/somerville/swag/display/SignUp.java [hunk 1]
```diff
@@ -1,15 +1,32 @@
...
     public JPanel root;
+    private JTextField textField5;
+    private JLabel lblAddressLineTwo;
+    private JLabel lblPhoneNumber;
+    private JLabel lblAddressLineOne;
+    private JLabel lblConfirmPassword;
+    private JLabel lblPassword;
+    private JLabel lblEmailAddress;
+    private JLabel lblSurname;
+    private JLabel lblForename;
```
Comment (sgoudham) [inline src/main/java/org/somerville/swag/display/SignUp.java:29]: Nice use of readable variable names clearly defining that it is a label

File: src/main/java/org/somerville/swag/display/SignUp.form [hunk 2]
```diff
@@ -1,59 +1,155 @@
...
+          </grid>
+        </constraints>
+        <properties/>
+      </component>
+      <component id="56210" class="javax.swing.JLabel" binding="lblAddressLineTwo">
+        <constraints>
+          <grid row="6" column="0" row-span="1" col-span="1" vsize-policy="0" hsize-policy="0" anchor="8" fill="0" indent="0" use-parent-layout="false"/>
+        </constraints>
+        <properties>
+          <text value="2nd Line of Address"/>
```
Comment (RobertRanson) [inline src/main/java/org/somerville/swag/display/SignUp.form:152]: This is looking great! I really like the '*' marking madatory feilds

**Review by 2013884f — APPROVED** [review]
Sign up form looks great

**Review by sgoudham — APPROVED** [review]

File: src/main/java/org/somerville/swag/display/SignUp.java [hunk 3]
```diff
@@ -1,15 +1,32 @@
...
```
Comment (RobertRanson) [inline src/main/java/org/somerville/swag/display/SignUp.java:30]: Really good naming conventions

**Review by RobertRanson — APPROVED** [review]

**Review by AlanRobinson1337 — COMMENTED** [review]
Greatly improved sign up form!

**Review by AlanRobinson1337 — APPROVED** [review]

**Pull request merged**